### PR TITLE
Added automatic pushing to Quay

### LINKED ISSUE #82: Automate timbot container build and publish to quay.io

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Fixes #82

### CODE REVIEW

File: .github/workflows/quay.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    env:
+      header: "Content-Type:application/json"
+      payload: "{\"commit\":\"$GITHUB_SHA\",\"ref\":\"$GITHUB_REF\",\"default_branch\":\"master\"}"
+      url: "${{ secrets.QUAY_WEBHOOK_URL }}"
+    steps:
+    - name: update container on quay.io
+      run: curl -H $header -d $payload $url
```
Comment (timflannagan) [inline .github/workflows/quay.yml]: I think we should still deflate the header environment variable and just hardcode the request header here.

Comment (timflannagan) [reply .github/workflows/quay.yml]: Also, I'm almost positive you can get the payload variable working by single-quoting it:

```suggestion
      run: curl -H $header -d '$payload' $url
```

Comment (nathan-weinberg) [reply .github/workflows/quay.yml]: Nope, wasn't the problem

File: .github/workflows/quay.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      header: "Content-Type:application/json"
+      payload: "{\"commit\":\"${{ github.sha }}\",\"ref\":\"${{ github.ref }}\",\"default_branch\":\"master\"}"
+      url: "${{ secrets.QUAY_WEBHOOK_URL }}"
+    steps:
+    - name: debug
+      run: |
+        echo $header
+        echo $payload
+    - name: update container on quay.io
+      run: curl -H $header -d $payload $url
```
Comment (timflannagan) [inline .github/workflows/quay.yml]: You may want to add some [built-in bash set options](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html) so we explicitly fail if there are things like pipe failures or undefined variables.

```suggestion
    - name: update container on quay.io
      run: |
        set -eu
        curl -H $header -d $payload $url
```

Comment (timflannagan) [reply .github/workflows/quay.yml]: You may need to add the `#! /bin/bash` shebang at the top too.

Comment (nathan-weinberg) [reply .github/workflows/quay.yml]: Bet will do

**Review by timflannagan — APPROVED** [review]

**Pull request merged**